cimgen/languages/modernpython/utils/export_template.mustache

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
1: in cgmes there are classes with eu profile
2: this won't allow correct serialisation of custom datatypes / classes

Reply:
What is wrong with this line?

Reply:
something went wrong, it was an addition to the other comment top of this :)